jscomp/syntax/ppx_entry.ml

```diff
@@ -255,8 +255,26 @@ let handle_core_type
   | _ -> super.typ self ty
 
 
-
-
+(*A very simple check if the string is a valid js regex*)
+let js_regex_checker s = 
+  let l = String.length s in
+  String.get s 0 = '/' && (String.get s (l-1) = '/' || String.get s (l-2) = '/')
+
```

Comment:
this looks very cool, can we move it into a stand alone file inside ext directory (with some comments)

you can test this function stand alone, see `ounit_tests` directory